docs/csharp/language-reference/compiler-options/advanced.md

```diff
@@ -85,6 +85,9 @@ When you specify [**DebugType**](code-generation.md#debugtype), the compiler cre
 
 ## PathMap
 
+> [!NOTE]
+Specifying **PathMap** may prevent breakpoints from working in local debug builds. **PathMap** should not be specified when building for the purposes of debugging.
```

Comment:
I'd say "will" instead of "may". 

I'd replace the second sentence with "Only set PathMap for production or [continuous integration builds](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#continuousintegrationbuild)."